server/conn_stmt.go

```diff
@@ -644,3 +644,20 @@ func (cc *clientConn) preparedStmt2String(stmtID uint32) string {
 	preparedAst := preparedObj.PreparedAst
 	return preparedAst.Stmt.Text() + sv.PreparedParams.String()
 }
+
+func (cc *clientConn) preparedStmt2StringNoArgs(stmtID uint32) string {
```

Comment:
... we could combine it with the above function.

Reply:
done, now let `preparedStmt2String` call `preparedStmt2StringNoArgs`

I didn't choose add a "flag param" to `preparedStmt2String`(which lead many lines be modified which make other reviewer unhappy) or add `opt func param` to `preparedStmt2String`(which is too heavy for this simple function)